documentation/developer_setup.md

```diff
@@ -96,7 +96,7 @@ Testing is built into the Go toolchain as well with the `testing` library. The t
 
 To get started first install [`docker`][get_docker]. Then to start the proxy, from the root of the repository, run the command `./eng/common/testproxy/docker-start-proxy.ps1 start`. This command will take care of pulling the pinned docker image and running it in the background.
 
-It is not required to run the test-proxy from within the docker container, but this is how the proxy is run in the Azure DevOps pipelines. If you would like to run the test-proxy in a different way check out the test-proxy [documentation][test_proxy_docs] for more information.
+It is not required to run the test-proxy from within the docker container, but this is how the proxy is run in the Azure DevOps pipelines. If you would like to run the test-proxy in a different the test-proxy [documentation][test_proxy_docs] has more information.
```

Comment:
The end of this sentence is missing a word.  "...run the test-proxy in a different the test-proxy..."